### PR TITLE
feat(Android): update getting started and feature pages

### PR DESCRIPTION
**Pull request by kahest**
## DESCRIBE YOUR PR
Updates Android getting started page to use new product selector. 

This also includes several smaller improvements to streamline the page and the navigation:
- move recent feature highlights to features page
- remove some superfluous/old notes that are not important enough any more to call out on the first landing page
- move some callouts and new items to next steps section
- refactor file structure so that there's no platform-includes anymore

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [x] ideally June 11, latest June 20 :) 

## SLA

- Teamwork makes the dream work, so please add a reviewer to your PRs.
- Please give the docs team up to 1 week to review your PR unless you've added an urgent due date to it.
Thanks in advance for your help!

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [x] Checked Vercel preview for correctness, including links
- [x] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [x] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## EXTRA RESOURCES

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

File: docs/platforms/android/index.mdx [hunk 1]
```diff
@@ -12,36 +12,106 @@ keywords:
...
+    'performance',
+    'profiling',
+  ]}
+/>
 
-<PlatformContent includePath="getting-started-install" />
+Sentry captures data by using an SDK within your application's runtime. These are platform-specific and allow Sentry to have a deep understanding of how your application works.
 
-## Configure
+In addition to capturing errors, you can monitor interactions between multiple services or applications by [enabling tracing](https://docs.sentry.io/concepts/key-terms/tracing/). You can also collect and analyze performance profiles from real users with [profiling](https://docs.sentry.io/product/profiling/). To enable tracing and/or profiling, click the corresponding checkmarks to get the code snippets.
```
Comment (markushi) [inline docs/platforms/android/index.mdx:35]:
```suggestion
In addition to capturing errors, you can monitor interactions between multiple services or applications by [enabling tracing](https://docs.sentry.io/concepts/key-terms/tracing/). You can also collect and analyze performance profiles from real users with [profiling](https://docs.sentry.io/product/profiling/). To enable tracing and/or profiling, click the corresponding checkmarks to customize the code snippets.
```

**Review by markushi — APPROVED** [review]
Nice, lgtm!

File: docs/platforms/android/index.mdx (hunk 1 above)
Comment (kahest) [reply docs/platforms/android/index.mdx:35]: IMO both makes sense - let's keep `get` for now as I copied that verbatim from Liza's suggestion on the Apple docs PR.

File: docs/platforms/android/features/index.mdx [hunk 2]
```diff
@@ -6,6 +6,17 @@ description: "Learn about the features of Sentry's Android SDK."
 
 Sentry's Android SDK enables automatic reporting of errors and exceptions, and identifies performance issues in your application. The below, is a list of features that are available as part of this SDK.
 
+<Expandable title="Already using Sentry for Android? Here are some recent highlights">
+
+We released <PlatformLink to="/performance/instrumentation/perf-v2/">Performance-V2</PlatformLink>
+, a set of experimental features to give you more details about app start performance and frame delay on a span level.
```
Comment (vivianyentran) [inline docs/platforms/android/features/index.mdx]:
```suggestion
We released <PlatformLink to="/performance/instrumentation/perf-v2/">Performance-V2</PlatformLink>,
a set of experimental features to give you more details about app start performance and frame delay on a span level.
```

File: docs/platforms/android/index.mdx (hunk 1 above)
Comment (vivianyentran) [inline docs/platforms/android/index.mdx:35]:
```suggestion
In addition to capturing errors, you can monitor interactions between multiple services or applications by [enabling tracing](https://docs.sentry.io/concepts/key-terms/tracing/). You can also collect and analyze performance profiles from real users with [profiling](https://docs.sentry.io/product/explore/profiling/). To enable tracing and/or profiling, click the corresponding checkmarks to get the code snippets.
```
updated profiling link

**Review by vivianyentran — APPROVED** [review]
Thanks for updating this!

**Pull request merged**